README.md

```diff
@@ -91,7 +90,14 @@ config :opentelemetry,
   processors: [
     otel_batch_processor: %{
       exporter:
-        {OpenTelemetry.Honeycomb.Exporter, write_key: "REPLACE_ME", dataset: "REPLACE_ME"}
+        {:opentelemetry_exporter,
+         %{
+           endpoints: [{:https, 'api.honeycomb.io', 443, '/v1/traces'}],
```

Comment:
This is actually super great, we can remove the rather problematic/old Honeycomb library and use default exporter, honeycomb will accept it.